data/tlg2035/tlg003/__cts__.xml

```diff
@@ -0,0 +1,13 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2035" xml:lang="grc" urn="urn:cts:greekLit:tlg2035.tlg003">
+  <ti:title xml:lang="lat">De decretis Nicaenae synodi</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg2035.tlg003.opp-grc1" workUrn="urn:cts:greekLit:tlg2035.tlg003">
+    <ti:label xml:lang="lat">De decretis Nicaenae synodi (Chapters 41 and 42)</ti:label>
+    <ti:description xml:lang="mul">Athanasius, De decretis Nicaenae synodi, Gelasius Kirchengeschichte, Loeschke and Heinemann,
+      Hinrichs, 1918</ti:description>
+  </ti:edition>
+  <ti:edition urn="urn:cts:greekLit:tlg2035.tlg003.opp-lat1" workUrn="urn:cts:greekLit:tlg2035.tlg003">
```

Comment:
@AlisonBabeu This one, I think, should be `<ti:translation>` instead of `<ti:edition>` since the original language of the work was Greek. Make sure to add an `xml:lang="lat"` attribute to the `<ti:translation>` element as well.